.github/actions/create-package/create-package.sh

```diff
@@ -71,6 +46,43 @@ wayland-decoration-client,wayland-graphics-integration-client,wayland-shell-inte
   echo "::endgroup::"
 }
 
+bundle_package_linux_linuxdeployqt() {
+    ${BUILD_CMD} install -Dm755 "/usr/lib/x86_64-linux-gnu/gstreamer1.0/gstreamer-1.0/gst-plugin-scanner" \
+      "Pencil2D/usr/lib/gstreamer1.0/gstreamer-1.0/gst-plugin-scanner"
+    local gst_executables="-executable=Pencil2D/usr/lib/gstreamer1.0/gstreamer-1.0/gst-plugin-scanner"
+    for plugin in adpcmdec alsa app audioconvert audioparsers audioresample \
+        autodetect coreelements gsm id3demux jack mpg123 mulaw playback \
+        pulse typefindfunctions wavparse apetag; do
+      ${BUILD_CMD} install -Dm755 "/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgst${plugin}.so" \
+        "Pencil2D/usr/lib/gstreamer-1.0/libgst${plugin}.so"
+      gst_executables="${gst_executables} -executable=Pencil2D/usr/lib/gstreamer-1.0/libgst${plugin}.so"
+    done
+    ${BUILD_CMD} curl -fsSLO https://github.com/probonopd/linuxdeployqt/releases/download/continuous/linuxdeployqt-continuous-x86_64.AppImage
+    ${BUILD_CMD} chmod 755 linuxdeployqt-continuous-x86_64.AppImage
+    local update_info="" # Currently no appimageupdate support for nightly builds
+    if [ $IS_RELEASE = "true" ]; then
+      update_info="-updateinformation=\"gh-releases-zsync|${GITHUB_REPOSITORY/\//|}|latest|pencil2d-linux-amd64-*.AppImage.zsync\""
+    fi
+    ${BUILD_CMD} bash -c "LD_LIBRARY_PATH=\"\${LD_LIBRARY_PATH}:/usr/lib/x86_64-linux-gnu/pulseaudio\" \
+      ./linuxdeployqt-continuous-x86_64.AppImage \
+      Pencil2D/usr/share/applications/org.pencil2d.Pencil2D.desktop \
+      -executable=Pencil2D/usr/plugins/ffmpeg \
+      ${gst_executables} \
+      -extra-plugins=platforms/libqwayland-egl.so,platforms/libqwayland-generic.so,platforms/libqwayland-xcomposite-egl.so,platforms/libqwayland-xcomposite-glx.so,wayland-decoration-client,wayland-graphics-integration-client,wayland-shell-integration \
+      ${update_info} \
+      -appimage"
+}
+
+bundle_package_linux_linuxdeploy() {
+${BUILD_CMD} curl -fsSL https://github.com/linuxdeploy/linuxdeploy/releases/download/continuous/linuxdeploy-x86_64.AppImage -o /usr/local/bin/linuxdeploy
+${BUILD_CMD} curl -fsSL https://github.com/linuxdeploy/linuxdeploy-plugin-qt/releases/download/continuous/linuxdeploy-plugin-qt-x86_64.AppImage -o /usr/local/bin/linuxdeploy-plugin-qt
+${BUILD_CMD} curl -fsSL https://github.com/linuxdeploy/linuxdeploy-plugin-appimage/releases/download/continuous/linuxdeploy-plugin-appimage-x86_64.AppImage -o /usr/local/bin/linuxdeploy-plugin-appimage
+${BUILD_CMD} curl -fsSL https://raw.githubusercontent.com/linuxdeploy/linuxdeploy-plugin-gstreamer/refs/heads/master/linuxdeploy-plugin-gstreamer.sh -o /usr/local/bin/linuxdeploy-plugin-gstreamer
+${BUILD_CMD} chmod 755 /usr/local/bin/linuxdeploy /usr/local/bin/linuxdeploy-plugin-qt /usr/local/bin/linuxdeploy-plugin-appimage /usr/local/bin/linuxdeploy-plugin-gstreamer
+export QMAKE=/usr/bin/qmake6
+${BUILD_CMD} linuxdeploy --appdir Pencil2D --plugin qt --plugin gstreamer --output appimage
```

Comment:
Would you mind adding indentation to this function like others? or any reason the spaces cannot be placed there?